enhancements/20220720-alertmanager.md

```diff
@@ -0,0 +1,303 @@
+# Harvester supports `Alertmanager`
+
+Customers wish to enable Prometheus `Alertmanager` in Harvester with the ability to configure in UI and API interfaces.
+
+## Summary
+.
+
+### Related Issues
+
+https://github.com/harvester/harvester/issues/2517
+
+## Motivation
+
+`Alert` is the very important information for monitoring one running system. Customers can integrate it with other systems to have a well defined lifecycle management.
+
+### Goals
+
+Support alertmanager.
+
+Support config various types of receivers.
+
+Support customized alert rules definition.
+
+Support customized template files to format alerts
+
+### Non-goals [optional]
+
+What is out of scope for this enhancement? Listing non-goals helps to focus discussion and make progress.
+
+## Proposal
+
+The Harvester `Alertmanager` will be achieved via `rancher-monitoring`.
+
+https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-monitoring/100.1.0%2Bup19.0.3
+
+The `Alertmanager` is disabled by default in Harvester as of v1.0.2.
+
+https://github.com/harvester/harvester-installer/blob/master/pkg/config/templates/rancherd-13-monitoring.yaml#L33
+
+```
+      alertmanager:
+        enabled: false
+```
+
+The `Alertmanager` will be enabled, and the related features will be described in following stories.
+
```

Comment:
Also adding custom configuration or just this parameter? Do we plan to support all that is supported via Rancher-Monitoring helm chart configs?
Hint: we might have to include all the images that are part of the rancher version we deliver  - i.e. https://github.com/rancher/rancher/releases/download/v2.6.6/rancher-images.txt